### PR TITLE
#158458729 deployment to heroku.

### PR DESCRIPTION
**Pull request by solnsubuga**
**What does this PR do?**

This PR contains the configurations or settings for easing deploying of the application on heroku.

**Description**
Create a heroku app and add configurations in application to serve it well.

**How should this be tested manually?**

Visit the application link [here](https://health-checks-wits.herokuapp.com/)

**Checklist**

- [x] I have added configuration values in settings file to serve the app on heroku and configured whitenose lib in wsgi file to enable serving of static files.

- [x] This PR is based on latest changes on develop.

**Any pivotal tracker user stories?**
#158458729

### CODE REVIEW

**Review by RrNn — COMMENTED** [review]
Looks good.

**Review by RrNn — APPROVED** [review]

**Review by wasswa-derick — APPROVED** [review]
Seems good to me

File: hc/settings.py [hunk 1]
```diff
@@ -100,6 +100,15 @@
         }
     }
 
+from decouple import config
```
Comment (john555) [inline hc/settings.py]: These imports should go to the top of the file.

File: .gitignore [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
+.vscode
 hc.sqlite
 hc/local_settings.py
 static-collected
 
 .DS_Store
-static/.DS_Store
\ No newline at end of file
+static/.DS_Store
+hc-venv
```
Comment (john555) [inline .gitignore]: Add a newline at the end of the file.

**Comment by john555** [comment]
@solnsubuga Squash your commits. Ideally, you should not have more than 3 commits on your PR.

**Review by Yiga-fred — APPROVED** [review]
Good Job Solo

**Pull request merged**